### DISCUSSION TITLE
Client config by renaming rustdesk .exe is ignored

### DISCUSSION THREAD

**Discussion by schertelk** [rustdesk/rustdesk#4291] [Q&A]
I renamed the rustdesk.exe (portable) to rustdesk-host=80.153.164.192,key=iwXXXXXXXjw=.exe
The "About" Window shows the correct host and key (see screenshot)
But the client trys to connect 155.138.247.159
From wireshark:
11281 630.742491 192.168.178.151 155.138.247.159 TCP 66 64348 → 21116 [SYN] Seq=0 Win=64240 Len=0 MSS=1460
WS=256 SACK_PERM

80.153.164.192 does not show up in wireshark.
Configuration with settings works.

![2023-05-06 16_58_10-RustDesk](https://user-images.githubusercontent.com/3830861/236631903-13984c1f-f54f-497e-8381-78e9a3f1bf9e.png)

**Comment by SergeyMy** [comment]
check the "nightly build"
https://github.com/rustdesk/rustdesk/releases/tag/nightly
1.1.9 hopelessly outdated

**Reply by schertelk** [reply]
wireshark shows me it still connects to 155.138.247.159
 
![2023-05-07 10_13_12-RustDesk](https://user-images.githubusercontent.com/3830861/236665941-1cf1e535-6dec-4f49-9830-e198a4adb79b.png)

**Reply by schertelk** [reply]
Now it works....
First  downloaded the .exe, renamed and started it - result see above
On the main page the client from the nightly build told it is oudated and I clicked the update button. The installer made an installation under windows programs. There was no installation on this system before, i tried only the .exe - i want a standalone solution for support similar to teamviewer quick connect.

Uninstalled the windows program. Result:
The client now shows a warning that because of UAC there can be problems and recommends to install the software on the system.
Ignored this - still works.

Very strange all that.

**Reply by schertelk** [reply]
OK, problem found.
In %appdata%\RustDesk existed thr configuration of a former installation, it was not deleted by the uninstall.
Cleaned it manually. 
The "portable" client is not "standalone" portable, because it uses and writes configuration data in %appdata%.

It would be nice to habe a real standalone client with easy configuration of own server. The commandline hack is only for windows. To configure an android client you have to type in the key, in a normal support case no option.
But this discussion already exists.
